### PR TITLE
[CI] adding CL application to centos configure

### PR DESCRIPTION
**Pull request by ddiezrod**
As a part of StructuralMechanicsApp was moved to CLApp we now compile this one too in altair, so I will add here it too. (In fact it seems there are some errors when compiling in centos ;) )

FYI @uksajja

### CODE REVIEW

**Comment by RiccardoRossi** [comment]
how can u add it if it has compilation errors?

**Comment by ddiezrod** [comment]
> how can u add it if it has compilation errors?

very easily, fixing the errors first :P  It looks like a very minor thing.

**Comment by ddiezrod** [comment]
It seems the error comes from one of your last changes @AlejandroCornejo could you take also a look, please?

**Comment by AlejandroCornejo** [comment]
HI! I'll adress this whenever I can, srry for the inconvenience :S

**Comment by AlejandroCornejo** [comment]
Weird, it gives problems from converting BoundedMatrix to ZeroMatrix....

File: applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.h [hunk 1]
```diff
@@ -99,15 +99,15 @@ class KRATOS_API(CONSTITUTIVE_LAWS_APPLICATION) AssociativePlasticDamageModel
...
-        BoundedMatrixType ComplianceMatrixIncrement = ZeroMatrix(VoigtSize,VoigtSize);
-        BoundedMatrixType ComplianceMatrix       = ZeroMatrix(VoigtSize,VoigtSize);
-        BoundedMatrixType ConstitutiveMatrix     = ZeroMatrix(VoigtSize,VoigtSize);
-        BoundedMatrixType TangentTensor          = ZeroMatrix(VoigtSize,VoigtSize);
-        BoundedVectorType PlasticFlow            = ZeroVector(VoigtSize);
-        BoundedVectorType PlasticStrain          = ZeroVector(VoigtSize);
-        BoundedVectorType PlasticStrainIncrement = ZeroVector(VoigtSize);
-        BoundedVectorType StrainVector           = ZeroVector(VoigtSize);
-        BoundedVectorType StressVector           = ZeroVector(VoigtSize);
+        BoundedMatrixType ComplianceMatrixIncrement { ZeroMatrix(VoigtSize,VoigtSize) } ;
```
Comment (AlejandroCornejo) [inline applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.h:102]: Interesting... Why is it needed??

Comment (ddiezrod) [reply applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.h:102]: honestly I dont know, @pooyan-dadvand ?

**Comment by philbucher** [comment]
should I make the centos build required?
has been there for a while now and works stable

**Comment by philbucher** [comment]
ok the centos build is required now
Let me know if anyone has an issue with it

**Comment by ddiezrod** [comment]
centos is passing now, can you please approve it?

**Review by AlejandroCornejo — APPROVED** [review]

**Comment by uksajja** [comment]
Thank you all

File: applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.h (hunk 1 above)
Comment (pooyan-dadvand) [reply applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.h:102]: Here you are explicitly calling the constructor and seems that for some reason GCC 4.8 has less problem in finding the correct constructor.

**Pull request merged**